src/components/HURUmap/ChartFilter/useStyles.js

```diff
@@ -0,0 +1,52 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(({ typography, palette }) => ({
+  root: {},
+  inputLabel: {
+    position: "inherit",
+    marginBottom: typography.pxToRem(3),
+    "&.MuiInputLabel-formControl": {
+      transform: "inherit",
+    },
+  },
+  label: {
+    fontSize: typography.pxToRem(10),
+    fontWeight: "700",
+    color: "#666666",
+    textTransform: "uppercase",
+  },
+  list: {
+    paddingTop: 0,
+    paddingBottom: 0,
+    "& li": {
+      fontWeight: 200,
+      paddingTop: 12,
+      paddingBottom: 12,
+    },
+    "& li.Mui-selected": {
+      fontWeight: "bold",
+    },
+  },
+  select: {
+    backgroundColor: palette.background.paper,
+    marginTop: "0 !important",
+    minWidth: typography.pxToRem(200),
+    "&.MuiInput-underline": {
+      "&::before": {
+        borderBottom: 0,
+      },
+      "&::after": {
+        borderBottom: 0,
+      },
+    },
+  },
+  placeholder: {
+    marginLeft: typography.pxToRem(15),
+    color: "#959696",
```

Comment:
Isn't this supposed to be [`#666666`](https://xd.adobe.com/view/c00dbbc6-0de1-476d-bd2a-dd778c185a3a-955e/specs/) as well?